[validate] - Fixed an issue with empty .pack-ignore with newlines (#3620)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased
* Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with brackets that contains a dot at the end of them.

> * Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that was previously empty with newlines.

Reviewer: @ShirleyDenkberg , please review.
```suggestion
* Fixed an issue where the **validate -g** failed reading a `.pack-ignore` file that was previously only contained newlines.
```
